content/2017-04-24_pycon-cz-2017-changes-date-and-venue.md

```diff
@@ -0,0 +1,41 @@
+Title: PyCon CZ 2017 changes date and venue
+Date: 2017-04-24 20:09:00
+Modified: 2017-04-24 20:09:00
+Author: Honza Javorek
+Twitter: honzajavorek
+Gravatar: mail@honzajavorek.cz
+About: [Honza](http://honzajavorek.cz) volunteers in the PyCon CZ organization team. His responsibility is to promote the event on social media and to take care of "Public Relations" of the conference.
+
+
+In the past month, the [PyCon CZ conference](https://cz.pycon.org/2017/) **became homeless**. Because of some property issues, the venue we originally announced suddenly became unavailable for any events.
+
+That has put our team into quite unpleasant situation - instead of working hard on the other tasks necessary for the conference to happen, we had to go back to the beginning and start looking for a new place. In a very tight schedule. After several weeks of contacting, visiting, and considering almost 50 venues, we're happy to present you with the replacement: **MeetFactory**!
+
+![MeetFactory from inside]({filename}/images/meetfactory6_music_hall_hr.jpg)
+
+**[MeetFactory](http://www.meetfactory.cz/en/)** is a non-profit **international center for contemporary art** founded in 2001 by the artist [David Černý](https://en.wikipedia.org/wiki/David_%C4%8Cern%C3%BD). It is in a very unique, **industrial building** in Prague's quarter Smíchov. It usually hosts music, theatre, gallery and the artist-in-residence program. You can walk through the whole space [on Google Maps](https://www.google.com/maps/@50.053037,14.408284,3a,75y,332h,90t/data=!3m5!1e1!3m3!1s5eyFEmazdNkAAAQfDU22GQ!2e0!3e2!6m1!1e1).
```

Comment:
hosts music events, theatre, gallery exhibitions, and residential artists program.